TestAssets/TestProjects/FSharpTestAppSimple/TestAppSimple.fsproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), testAsset.props))\testAsset.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
@enricosada Could you add a .NET Standard 2.0 library test as well please?  Also a .NET Standard 1.6 library test if it is possible to build such a beast with the F# support, and a range of  .NET Framework libraries too?